numpy/core/_methods.py

```diff
@@ -115,10 +115,7 @@ def _var(a, axis=None, dtype=None, out=None, ddof=0, keepdims=False):
     # Note that x may not be inexact and that we need it to be an array,
     # not a scalar.
     x = asanyarray(arr - arrmean)
-    if issubclass(arr.dtype.type, nt.complexfloating):
-        x = um.multiply(x, um.conjugate(x), out=x).real
-    else:
-        x = um.multiply(x, x, out=x)
+    x = um.multiply(x,um.conjugate(x),out=x).real
```

Comment:
This isn't quite the solution suggested by @eric-wieser  in https://github.com/numpy/numpy/issues/13110#issuecomment-473191750: rather the suggestion is to replace `issubclass(arr.dtype.type, nt.complexfloating)` with `not issubclass(arr.dtype.type, (nt.floating, nt.integer))` (or something like it, I didn't check naming details), i.e., we do want to keep the optimization for numbers we know are not complex (because `.conjugate` makes a copy; if it were just a read-only view this would not be an issue, but that's a separate question).

Reply:
Thank you @mhvk I have made the necessary changes in my latest commit. `not issubclass(arr.dtype.type, (nt.floating, nt.integer))` definitely helps. This ensures that optimisation for non complex numbers is kept intact.